hangups/default.conf

```diff
@@ -0,0 +1,11 @@
+[General]
+col-scheme: default
+
+[Keybindings]
+key-prev-tab: ctrl u
+key-next-tab: ctrl d
+key-close-tab: ctrl w
+key-quit: ctrl e
+key-menu: ctrl n
+key-up: k
+key-down: j
```

Comment:
`__main__.py` is currently loading `default.conf` from whatever the current directory is when hangups is launched, eg. if you're in `$HOME/hangups`, it will look for `$HOME/hangups/default.conf`

Would putting it in site-packages along with the code be useful to anyone? It seems like that wouldn't be discoverable.